doc/whats_new.rst

```diff
@@ -239,9 +239,17 @@ Bug fixes
      left `coef_` as a list, rather than an ndarray.
      :issue:`8160` by :user:`CJ Carey <perimosocordiae>`.
 
+
+   - Fix a bug where `sklearn.feature_extraction.FeatureHasher` mandatorily
```

Comment:
This is rst not md, so you want to use double-backticks. Best is to use the sphinx directives `:class:` or `:func:`. Look around in this file for examples.

Note this happens a few times below as well.